user-docs/latest/content-manager/writing-content.md

```diff
@@ -0,0 +1,63 @@
+| Enumeration | 1. Click the listbox. <br> 2. Choose an item from the list.                                                                                                                                                                                                                                                                                                                  |
+| Media       | 1. Click the media area. <br> 2. Choose an asset from the Media Library, or click the **Add more assets** button to add a new file to the Media Library. <br><br> 💡 It is possible to drag and drop the chosen file in the media area.                                                                                                                                        |
+| JSON        | Write your content, in JSON format, in the code textbox.                                                                                                                                                                                                                                                                                                                     |
+| UID         | Write a unique identifier in the textbox. A "Regenerate" button, displayed on the right of the box, allows to automatically generate a UID based on the content type name.                                                                                                                                                                                                   |
+
+### Components
+
+Components are a combination of several fields. Writing their content works exactly like for independant fields. They differ in their display, since the fields of a component are grouped together.
+
+![Writing content for a component](../assets/content-manager/edit-view_component.png)
+
+### Dynamic zones
+
+Dynamic zones are a combination of components, which themselves are composed of several fields. Writing the content of a dynamic zone requires additional steps in order to access the fields.
+
+1. Click on the ![icon add to content](../assets/content-manager/icon_add3.png) **Add to content** button.
+2. Choose a component available for the dynamic zone.
+3. Fill in the fields of the component.
+
+![Writing content for a dynamic zone](../assets/content-manager/edit-view_dynamic-zone1.png)
+![Writing content for a dynamic zone](../assets/content-manager/edit-view_dynamic-zone2.png)
+
+In contrary to regular fields and components, dynamic zones can also be reordered or deleted directly in the edit view, using buttons displayed in the top right corner of the dynamic zone area.
```

Comment:
This is incorrect information.
When you are having a "repeatable component", you can drag and drop each element.
Then in this section, you mentioned the reorder and delete 👇, I think it is worth it to add it also in the Component section.
Plus I would add the fact that a component can be repeatable or not and the rendering is a bit different.